igvc_navigation/launch/navigation_client.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+    <node name="navigation_client" pkg="igvc_navigation" type="navigation_client" output="screen">
+        <param name="reading_from_file" value="true"/>
```

Comment:
You should make this an [arg](http://wiki.ros.org/roslaunch/XML/arg), and add a corresponding argument to `navigation_simulation.launch`, so that you can control whether waypoints are used or not by simply doing
`roslaunch igvc_navigation navigation_simulation.launch waypoints:=true` or something